Package.swift

```diff
@@ -39,6 +39,17 @@ let javaIncludePath = "\(javaHome)/include"
   let javaPlatformIncludePath = "\(javaIncludePath)/win32"
 #endif
 
+func sjEnableBenchmarking() -> Bool {
+    if let value = ProcessInfo.processInfo.environment["SWIFT_JAVA_ENABLE_BENCHMARKING"] {
+        switch value {
+        case "TRUE", "YES", "1": return true
+        case "FALSE", "NO", "0": return false
+        default: break
+        }
+    }
+    fatalError("Please set 'SWIFT_JAVA_ENABLE_BENCHMARKING' to 'TRUE' or 'FALSE'")
+}
```

Comment:
We shouldn't force all clients to set an environment variable here. I think the best option here is to disable benchmarking on Windows until we find an appropriate solution.

Reply:
Is there a way to do that within the confines of package.swift? `#if os(...)` uses the build not the host.

Reply:
I'll repeat the request about the "opt out with an env var" on windows.

I don't understand what you meant by it being hard on windows. There is "set ...=..." and surely IDEs can set env when starting processes as well...? Can we do that instead?

Reply:
No, IDEs don't do very well with setting environment variables (at least VSCode in some cases and VS do not). The environment variable setting then permeates into everything and has different syntax in different places. It doesn't fully propagate between subshells, which adds additional surprises. I think that we should either fully drop the benchmarking or require setting the environment everywhere - it avoids any surprises.

Reply:
Then I propose we put the benchmarking into a sub-package that we add in a sub directory. Lemme try that.